queue-management-service/src/main/java/io/queuemanagement/api/interfaces/stream/kafkaconsumer/ConcertReservationConfirmMessageConsumer.java

```diff
@@ -0,0 +1,38 @@
+package io.queuemanagement.api.interfaces.stream.kafkaconsumer;
+
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.kafka.support.Acknowledgment;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.queuemanagement.api.interfaces.stream.payload.ConcertReservationConfirmMessagePayload;
+import io.queuemanagement.common.annotation.KafkaConsumer;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author : Rene Choi
+ * @since : 2024/07/11
+ */
+@Component
+@RequiredArgsConstructor
+@KafkaConsumer
+@Slf4j
+public class ConcertReservationConfirmMessageConsumer implements KafkaMessageConsumer{
+
+	private final ObjectMapper objectMapper;
+	private final ApplicationEventPublisher applicationEventPublisher;
+
+	@Override
+	public void handle(Message<String> message, Acknowledgment acknowledgment) {
+		log.info("Received message: {}", message.getPayload());
+		try {
+			applicationEventPublisher.publishEvent(objectMapper.readValue(message.getPayload(), ConcertReservationConfirmMessagePayload.class));
```

Comment:
```suggestion
			processingQueueFacade.completeToken(event.toCompletedTokenHandlingRequest());
```
모든 로직을 이벤트화시키는건 코드의 가독성을 해칠 위험이 있어요. 여기서는 직접 호출하는게 나아보여요
메인로직과 부가로직으로 봤을때 부가로직을 이벤트로 관심사를 분리하는게 좋아요.
여기선 메인로직이 토큰을 만료시키는 것 !